packages/vscode-extension/lib/inspector.js

```diff
@@ -0,0 +1,112 @@
+const { Dimensions, findNodeHandle } = require("react-native");
+const getInspectorDataForViewAtPoint = require("react-native/Libraries/Inspector/getInspectorDataForViewAtPoint");
```

Comment:
There's a reason why we use inline require for this method in wrapper.js

In short: since wrapper can be loaded out of normal order, we want to avoid a chain reaction of other modules being loaded that way. Hence, we should only load it when we need it (in getInspectorDataForCoordinates method) – at that point the whole bundle should be fully loaded and executed hence the order should no longer matter.

Reply:
thanks for this remark, I feel like I'm learning a lot about javascript in general working on this project:D 

I changed it back to inline require